dashboard/app/controllers/learning_goal_evaluations_controller.rb

```diff
@@ -4,15 +4,15 @@ class LearningGoalEvaluationsController < ApplicationController
   def create
     learning_goal_evaluation = LearningGoalEvaluation.new(learning_goal_evaluation_params)
     learning_goal_evaluation.teacher_id = current_user.id
+    learning_goal_evaluation.created_at = DateTime.current
```

Comment:
you shouldn't need this -- Rails will set `created_at` and `updated_at` automatically